scipy/stats/tests/test_stats.py

```diff
@@ -403,137 +403,6 @@ def test_len1(self):
-        res = fisher_exact([[0, 1], [3, 2]])[1]
-        assert_approx_equal(res, 1.0)
-        res = fisher_exact([[0, 2], [6, 4]])[1]
-        assert_approx_equal(res, 0.4545454545)
-        res = fisher_exact([[2, 7], [8, 2]])
-        assert_approx_equal(res[1], 0.0230141, significant=6)
-        assert_approx_equal(res[0], 4.0 / 56)
-
-    def test_precise(self):
-        # results from R
-        #
-        # R defines oddsratio differently (see Notes section of fisher_exact
-        # docstring), so those will not match.  We leave them in anyway, in
-        # case they will be useful later on. We test only the p-value.
-        tablist = [
-            ([[100, 2], [1000, 5]], (2.505583993422285e-001, 1.300759363430016e-001)),
-            ([[2, 7], [8, 2]], (8.586235135736206e-002, 2.301413756522114e-002)),
-            ([[5, 1], [10, 10]], (4.725646047336584e+000, 1.973244147157190e-001)),
-            ([[5, 15], [20, 20]], (3.394396617440852e-001, 9.580440012477637e-002)),
-            ([[5, 16], [20, 25]], (3.960558326183334e-001, 1.725864953812994e-001)),
-            ([[10, 5], [10, 1]], (2.116112781158483e-001, 1.973244147157190e-001)),
-            ([[10, 5], [10, 0]], (0.000000000000000e+000, 6.126482213438734e-002)),
-            ([[5, 0], [1, 4]], (np.inf, 4.761904761904762e-002)),
-            ([[0, 5], [1, 4]], (0.000000000000000e+000, 1.000000000000000e+000)),
-            ([[5, 1], [0, 4]], (np.inf, 4.761904761904758e-002)),
-            ([[0, 1], [3, 2]], (0.000000000000000e+000, 1.000000000000000e+000))
-            ]
-        for table, res_r in tablist:
-            res = stats.fisher_exact(np.asarray(table))
-            np.testing.assert_almost_equal(res[1], res_r[1], decimal=11,
-                                           verbose=True)
-
-    @pytest.mark.slow
-    def test_large_numbers(self):
-        # Test with some large numbers. Regression test for #1401
-        pvals = [5.56e-11, 2.666e-11, 1.363e-11]  # from R
-        for pval, num in zip(pvals, [75, 76, 77]):
-            res = stats.fisher_exact([[17704, 496], [1065, num]])[1]
-            assert_approx_equal(res, pval, significant=4)
-
-        res = stats.fisher_exact([[18000, 80000], [20000, 90000]])[1]
-        assert_approx_equal(res, 0.2751, significant=4)
-
-    def test_raises(self):
-        # test we raise an error for wrong shape of input.
-        assert_raises(ValueError, stats.fisher_exact,
-                      np.arange(6).reshape(2, 3))
-
-    def test_row_or_col_zero(self):
-        tables = ([[0, 0], [5, 10]],
-                  [[5, 10], [0, 0]],
-                  [[0, 5], [0, 10]],
-                  [[5, 0], [10, 0]])
-        for table in tables:
-            oddsratio, pval = stats.fisher_exact(table)
-            assert_equal(pval, 1.0)
-            assert_equal(oddsratio, np.nan)
-
-    def test_less_greater(self):
-        tables = (
-            # Some tables to compare with R:
-            [[2, 7], [8, 2]],
```

Comment:
It looks like these tables are no longer tested, or were they moved somewhere other than `test_less_greater`?

Reply:
These tables are included in the file `fisher_exact_results_from _r.py`, which includes all three alternatives for each table.